src/domain/home/form/types/FormTypes.ts

```diff
@@ -0,0 +1,12 @@
+export interface HomeFormTypes {
```

Comment:
well the interface name should stay as it is before you move
`HomeFormValues`

But the filename shoud be HomeFormTypes